istqb-ct-genai-complete-guide.md

```
@@ -0,0 +1,1675 @@
+> │   ┌──────────────────────────────────────────────────┐          │
+> │   │  CTFL v4.0（前提資格：必須）                        │          │
+> │   └──────────────────────────────────────────────────┘          │
+> │                                                                  │
+> │  🔑 キャリアパス：                                                  │
+> │  CT-GenAI → CTAL-TA / CTAL-TTA / CTAL-TM → Expert Level        │
+> └──────────────────────────────────────────────────────────────────┘
+
+### 0.2 CT-GenAI が生まれた背景
+
+> なぜ今 CT-GenAI が必要なのか？
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)